src/internal/header.rs

```diff
@@ -31,7 +36,7 @@ impl Header {
                 self.0,
                 tag as i32,
                 &mut td,
-                librpm_sys::headerGetFlags_e_HEADERGET_MINMEM,
+                librpm_sys::headerGetFlags_e_HEADERGET_ALLOC,
             )
         };
 
```

Comment:
@tarcieri On the line below, the error message doesn't seem to match the assert - I'm guessing this should be `assert_eq!()`?